src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -92,10 +92,14 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
           preferences.putAll(configuration.asMap(buildCustomConfPrefix(formId)))
       );
 
-      if (forms.someSelected() && (form.getConfPanel().isValid() && forms.allSelectedFormsHaveConfiguration()))
+      if (forms.someSelected() && (form.getConfPanel().isValid() && forms.allSelectedFormsHaveConfiguration())) {
+        form.setErrorMsg("No errors! Ready to export form(s)");
```

Comment:
Since this feedback won't change what the user is doing, I would remove it. Let's give feedback when we need the user to do something or if some process is really long.